app/services/post_importer.rb

```diff
@@ -3,7 +3,7 @@ def initialize(url)
     @url = url
   end
 
-  def import(board_id, importer_id, section_id: nil, status: Post::Status::COMPLETE, threaded: false)
+  def import(board_id, importer_id, section_id: nil, status: Post.statuses[:complete], threaded: false)
```

Comment:
I left the post scraper/importer code alone for now and let it keep using integers; possibly this should be refactored to use strings/symbols eventually? Do we care enough that I should do that refactoring now?